lib/tools/apk-utils.js

```diff
@@ -164,7 +164,9 @@ apkUtilsMethods.startApp = async function startApp (startAppOptions = {}) {
  */
 apkUtilsMethods.getFocusedPackageAndActivity = async function getFocusedPackageAndActivity () {
   log.debug('Getting focused package and activity');
-  const cmd = ['dumpsys', 'window', 'windows'];
+  const apiLevel = await this.getApiLevel();
+  const dumpsysArg = apiLevel >= 29 ? 'displays' : 'windows';
```

Comment:
there should be a comment about why it is necessary. Also, we need to find other places in the code where this call is used and probable isolate them into a separate method call